_includes/core/footer.html

```diff
@@ -0,0 +1,20 @@
+<footer>
+  <div class="inner-footer">
+    <p>© 2015 Microsoft, Inc.</p>
```

Comment:
Should be (C) .NET Foundation (no year).  FYI, recommended copyright under MSFT would be (C) Microsoft Corp (again no year)